AmazonEmailServiceConfiguration.pas

```diff
@@ -0,0 +1,55 @@
+unit AmazonEmailServiceConfiguration;
+
+interface
+
+type
+  TAmazonEmailServiceConfiguration = class
+  private
+    function GetEnvVarValue(const VarName: string): string;
+  public
+    procedure GetFromEnvironment(var AwsSESEndpoint, AwsSESAccessKey, AwsSESSecretKey: string);
+  end;
+
+const
+  AWS_SES_REGION_ENDPOINT = 'AWS_SES_REGION_ENDPOINT';
+  AWS_SES_ACCESS_KEY_ID = 'AWS_SES_ACCESS_KEY_ID';
+  AWS_SES_SECRET_ACCESS_KEY = 'AWS_SES_SECRET_ACCESS_KEY';
```

Comment:
It is best to follow the same variables names used in official libraries as I have described in #4.